src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react'
 import styles from 'components/DropdownMenu/DropdownMenu.module.scss'
 
+// TODO - clean up prop types for triggerEl
+
 type PropTypes = {
   children: React.ReactNode
-  menuIcon: React.ReactNode
-  ariaLabel: string
-  onMenuClick: () => void
+  menuIcon?: React.ReactNode
+  ariaLabel?: string
+  onMenuClick?: () => void
   isActive: boolean
-  dropdownRef: React.RefObject<HTMLDivElement>
+  dropdownRef?: React.RefObject<HTMLDivElement>
+  triggerEl?: React.ReactNode
```

Comment:
@abbyoung I made some changes to the DropdownMenu to allow passing in an entire element to use as the open/close toggle, because I needed to be able to style this button and make sure it had certain attributes (like `id`). This leaves the prop types in a weird state because if someone passes in a `triggerEl` that means they don't need to pass in props like `menuIcon, ariaLabel, onMenuClick`... we can either update this component so that it's compatible with either method, _or_ require that `triggerEl` is always passed in. What do you think?

Reply:
i know we briefly discussed in person but for the record: i like toggleEl much better 👍